media/stylus/components/share.styl

```diff
@@ -12,32 +13,78 @@
     li {
         display: inline-block;
         margin: 0;
-        padding: 0;
+        bidi-value(padding, 0, 0); /* to over-ride default bidi-value) */
     }
-}
 
-.share-link {
-    border: 1px solid rgba-fallback(rgba(255,255,255,0.2));
-    border-radius: 2px;
-    box-shadow: none;
-    color: #fff;
-    bidi-value(margin, 0px 6px 6px 0px, 0px 0px 6px 6px);
-    padding: 3px;
+    &.show {
+        display: block;
+    }
+
+
+    /*
+    Inside TOC
+    -------------------------------------------------------------- */
+    .toc & {
+        background-color: #fff;
+        margin: 0px -15px -5px;
+        padding: 5px 0 15px 0;
+        top: 15px;
+
+        @media $media-query-small-desktop {
+            background-color: transparent;
+            top: 0;
+        }
 
-    &.facebook {
-        background-color: rgb(59, 89, 152);
+        /* hide title if there isn't enough space for it in TOC */
+        @media all and (min-width:1025px) and (max-width: 1400px),
```

Comment:
It's one pixel off, but we could try the similar `$media-query-tablet` for a `min-width` of `1024px`.